### DISCUSSION TITLE
Contacts example app

### DISCUSSION THREAD

**Discussion by Anan-Saadi** [isar/isar#302] [Ideas]
Isar docs shows there is a contacts example app but it has been saying "coming soon" since forever, and as far as I can tell there is no new project being worked on in the [samples](https://github.com/isar/samples) repo, so can I build it myself and add it to the repo ?

**Comment by Anan-Saadi** [comment]
Hey @leisim, i don't know if you saw this but what do you think ?

**Reply by simc** [reply]
@Anan-Saadi Hey, sorry I completely missed this thread! That would be amazing. I wanted to write a whole step by step tutorial for the contacts app. If you'd like to give it a go or just write the app that would be greatly appreciated 🥰

**Reply by Anan-Saadi** [reply]
@leisim What should the app do exactly, I assume it should utilize most of isar features right?
